engine/apps/user_management/models/organization.py

```diff
@@ -133,6 +133,7 @@ def _get_subscription_strategy(self):
 
     gcom_token = mirage_fields.EncryptedCharField(max_length=300, null=True, default=None)
     gcom_token_org_last_time_synced = models.DateTimeField(null=True, default=None)
+    gcom_contract_type = models.CharField(max_length=300, null=True, default=None)
```

Comment:
May want to use `gcom_plan` as the the field name since this is what the field is called in instance info, or is this going to reference different information than that?